tests/unit/utils/test_utils_conferences.py

```diff
@@ -120,3 +120,34 @@ def test_render_contributions():
     result = render_contributions(hits)
 
     assert expected == result
+
+
+def test_render_contributions_handles_unicode():
+    hits = Response({
+        'hits': {
+            'hits': [
+                {
+                    '_type': 'hep',
+                    '_source': {
+                        'control_number': 1427573,
+                        'titles': [
+                            {'title': u'Storage Ring Based EDM Search — Achievements and Goals'},
+                        ],
+                    },
+                },
+            ],
+            'total': 1,
+        },
+    }).hits
+
+    expected = ([
+        [
+            u"<a href='/literature/1427573'>Storage Ring Based EDM Search — Achievements and Goals</a>",
+            u'\n  \n  \n\n\n  \n',
```

Comment:
Please, make this test to ignore whitespaces. In this there won't be a need to amend it each time someone re-indent the template.